sys/mips/mips/trap.c

```diff
@@ -969,7 +967,11 @@ trap(struct trapframe *trapframe)
 				ucode = SEGV_ACCERR;
 			else
 				ucode = SEGV_MAPERR;
+#ifdef CPU_CHERI
```

Comment:
I wonder if we want a TF_PCC macro that expands to pcc or pc instead of all these ifdefs?